frontend/src/components/TableSortIcon.vue

```diff
@@ -25,7 +25,7 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from "@nuxtjs/composition-api"
+import { defineComponent } from "vue"
```

Comment:
The PR description mentions that `defineComponent` should be imported from `@nuxtjs/composition-api`.

Reply:
`defineComponent` from "vue" works well for components, but it breaks the `useMeta` in pages: the page title is not set anymore.
I think it might be because `defineComponent` has several overrides in "vue", and the wrong one gets picked up.